ivy_tests/test_ivy/test_frontends/test_torch/test_loss_functions.py

```diff
@@ -200,16 +200,19 @@ def test_torch_binary_cross_entropy(
     size_average=st.booleans(),
     reduce=st.booleans(),
     reduction=st.sampled_from(["mean", "none", "sum", None]),
-    dtype_and_pos_weight=helpers.array_or_none(
-        array_dtype="float",
-        min_value=0,
-        max_value=10,
-        allow_inf=False,
-        exclude_min=True,
-        exclude_max=True,
-        min_num_dims=1,
-        max_num_dims=1,
-        min_dim_size=2,
+    dtype_and_pos_weight=st.one_of(
+        helpers.dtype_and_values(
+            array_dtype="float",
+            min_value=0,
+            max_value=10,
+            allow_inf=False,
+            exclude_min=True,
+            exclude_max=True,
+            min_num_dims=1,
+            max_num_dims=1,
+            min_dim_size=2,
+        ),
+        st.just([None], [None]),
```

Comment:
`st.just` takes only 1 parameter, this should be a list instead.

Reply:
Thanks for the pointer! So I'd assume we want `st.just([None])`, is that right?